src/win.c

```diff
@@ -989,6 +986,24 @@ bool win_should_fade(session_t *ps, const struct managed_win *w) {
 	return ps->o.wintype_option[w->window_type].fade;
 }
 
+/**
+ * Determine if a window should animation.
```

Comment:
Bad english: "Determine if a window should animation" -> "Determine if a window should animate".